packages/cli/src/commands/source/push/delete.ts

```diff
@@ -0,0 +1,165 @@
+import {Command, flags} from '@oclif/command';
+import {Source} from '@coveo/push-api-client';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+import {
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import dedent from 'ts-dedent';
+import {
+  buildAnalyticsFailureHook,
+  buildAnalyticsSuccessHook,
+} from '../../../hooks/analytics/analytics';
+import {green, red} from 'chalk';
+// eslint-disable-next-line node/no-extraneous-import
+import {AxiosResponse} from 'axios';
+
+interface ErrorFromAPI {
+  response: {
+    status: number;
+    data: {
+      errorCode: string;
+      message: string;
+    };
+  };
+}
+
+export default class SourcePushDelete extends Command {
+  public static description =
+    'Delete one or multiple documents in a given push source. See https://docs.coveo.com/en/171 and https://docs.coveo.com/en/131';
+
+  public static flags = {
+    deleteOlderThan: flags.string({
+      char: 'd',
+      exclusive: ['delete'],
+      description:
+        'Delete old items, using either an ISO 8601 date or a Unix timestamp',
+      helpValue: '2000-01-01T00:00:00-06:00 [ OR ] 946702800',
```

Comment:
Hm I don't get the [ ] around the OR.

At first I thought it might be some odd syntax like, `2000-01-01T00:00:00-06:00 [` being 'starting from 2000-01-01T00:00:00-06:00, excluding it' and `] 946702800` being until 946702800, excluding it.

Reply:
I'm trying to explain in the help value that multiple date format are accepted, either an ISO date or a timestamp.

The output when you `--help` is something like: 

```
--deleteOlderThan=2000-01-01T00:00:00-06:00 [ OR ] 946702800
```

Open to suggestion !

Reply:
👍 I would just put OR but not the `[` `]`